### PR TITLE
Add Note About Using Hover With Animation

### PR DESCRIPTION
**Pull request by Yarob50**
Hi guys, thanks for your great work,
while I was trying Tailwind Elements as a preparation for the Youtube video, I found some missing info that I really needed to understand how things work. One of them was this one.

In the example of `hover` I needed to know which classes are the ones responsible for making the animation (transition), I understand that this is part of tailwind docs, but I find it really useful to be mentioned in your docs as well.

I will be more than happy to contribute to this nice package today and maybe more in the future, with that being said, If you disagree with my suggestion/change, this is totally fine. 
Just explain your opinion to me from your point of view.

👌🏻👏🏻

### CODE REVIEW

File: site/content/docs/standard/content-styles/hover-effects/index.html [hunk 1]
```diff
@@ -50,6 +50,32 @@
   {{< /twsnippet/demo >}}
 
 
+  <!-- Section: Note about usage with animation  -->
+  <section>
+    <h3>Showing the animation effect</h3>
```
Comment (smolenski-mikolaj) [inline site/content/docs/standard/content-styles/hover-effects/index.html]: Please add the following class to h3 element to be consistent with the rest of the document: `mb-3 text-2xl font-semibold leading-normal`.

File: site/content/docs/standard/content-styles/hover-effects/index.html [hunk 2]
```diff
@@ -50,6 +50,32 @@
   {{< /twsnippet/demo >}}
 
 
+  <!-- Section: Note about usage with animation  -->
```
Comment (smolenski-mikolaj) [inline site/content/docs/standard/content-styles/hover-effects/index.html]: Please move the whole section before this ending tag in the docs:

```
</section>
<!-- Section: Basic example -->
```

It should be added after the code snippet.

File: site/content/docs/standard/content-styles/hover-effects/index.html [hunk 3]
```diff
@@ -50,6 +50,32 @@
...
+  <section>
+    <h3>Showing the animation effect</h3>
+    <p class="mb-3">
+      As you can see in the above example, you can add any animation to any
+      element on the page by using the following classes:
+    </p>
+    <ul>
+      <li>
+        <code>transition</code> : Used to animate the chaged properties on
```
Comment (smolenski-mikolaj) [inline site/content/docs/standard/content-styles/hover-effects/index.html]: Please change `Used` to `used`.

File: site/content/docs/standard/content-styles/hover-effects/index.html [hunk 4]
```diff
@@ -50,6 +50,32 @@
...
+        hover.
+      </li>
+
+      <li>
+        <code>duration-x</code> : Used to define the duration of the animation
```
Comment (smolenski-mikolaj) [inline site/content/docs/standard/content-styles/hover-effects/index.html]: Please change `Used` to `used`.

File: site/content/docs/standard/content-styles/hover-effects/index.html [hunk 5]
```diff
@@ -50,6 +50,32 @@
...
+        where <code>x</code> is the time in melli seconds; ex:
```
Comment (smolenski-mikolaj) [inline site/content/docs/standard/content-styles/hover-effects/index.html]: Please change `melli seconds` to `milliseconds`. Also, please add the link to Tailwind's duration docs to help our users to find the built-in classes. Here's the url: https://tailwindcss.com/docs/transition-duration.

File: site/content/docs/standard/content-styles/hover-effects/index.html [hunk 6]
```diff
@@ -50,6 +50,32 @@
...
+        <code>duration-300</code>.
+      </li>
+    </ul>
+    <p>
+      then, you should be able to add any type of animation by changing the
```
Comment (smolenski-mikolaj) [inline site/content/docs/standard/content-styles/hover-effects/index.html]: Please change `then` into `Then`.

File: site/content/docs/standard/content-styles/hover-effects/index.html [hunk 7]
```diff
@@ -50,6 +50,32 @@
...
+      property on hover by using the class: <code>hover:tw-class-name</code> ex:
```
Comment (smolenski-mikolaj) [inline site/content/docs/standard/content-styles/hover-effects/index.html]: Please change `ex` to `e.g.`. Also please change `hover:tw-class-name` to `hover:[tw-class-name]` to make sure users will not use the actual class name from this example.

File: site/content/docs/standard/content-styles/hover-effects/index.html [hunk 8]
```diff
@@ -50,6 +50,32 @@
...
+  <section>
+    <h3>Showing the animation effect</h3>
+    <p class="mb-3">
+      As you can see in the above example, you can add any animation to any
+      element on the page by using the following classes:
+    </p>
+    <ul>
+      <li>
+        <code>transition</code> : Used to animate the chaged properties on
+        hover.
```
Comment (smolenski-mikolaj) [inline site/content/docs/standard/content-styles/hover-effects/index.html]: Please change `.` to `,`

**Comment by smolenski-mikolaj** [comment]
@Yarob50 thanks for your pull request. The text looks very valuable. We will gladly put it on the website, after editing it in relation to the proposed changes.

File: site/content/docs/standard/content-styles/hover-effects/index.html (hunk 7 above)
Comment (Yarob50) [reply site/content/docs/standard/content-styles/hover-effects/index.html]: good one 👌🏻

**Comment by Yarob50** [comment]
@smolenski-mikolaj thanks for your response and valuable feedback,
I made the changes as proposed, please check them out and let me know if I missed anything or if there is any other thing to be enhanced/changed.

**Pull request merged**